scripts/remove_after_use/end_prereg_challenge.py

```diff
@@ -19,11 +19,15 @@
 def main(dry_run=True):
 
     switch = Switch.objects.get(name='osf_preregistration')
+    logger.info('Setting {} switch to active.'.format(switch.name))
     switch.active = True
     prereg_challenge_schema = RegistrationSchema.objects.get(name='Prereg Challenge')
+    logger.info('Setting {} schema to inactive'.format(prereg_challenge_schema.name))
     prereg_challenge_schema.active = False
     prereg_schema = RegistrationSchema.objects.get(name='OSF Preregistration')
+    logger.info('Setting {} switch to active.'.format(prereg_schema.name))
```

Comment:
Typo: "switch" -> "schema"

Reply:
Super nitpicky nit: punctuation should be consistent (2 statements have periods, 1 doesn't). I think you can omit the periods.